server/models/Review.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const Message = require('./Message');
+
+const { Schema, ObjectId } = mongoose;
+
+const reviewSchema = new Schema({
+  requesterId: { type: ObjectId, required: true },
+  reviewerId: ObjectId,
+  declinedIds: [ObjectId],
+  title: String,
+  date: { type: Date, default: Date.now },
+  language: { type: String, required: true },
+  code: { type: String, required: true },
+  comments: String,
```

Comment:
I don't love the repeat of code, comments, etc. -> maybe the first message could be the code and comments mentioned here, and you just keep title and language attached to the review schema